attributes/defaults.rb

```diff
@@ -73,7 +73,7 @@
 default['mysqld']['my.cnf']['mysqld']['log_error'] = '/var/log/mysql/error.log'
 default['mysqld']['my.cnf']['mysqld']['expire_logs_days'] = 10
 default['mysqld']['my.cnf']['mysqld']['max_binlog_size'] = '100M'
-default['mysqld']['my.cnf']['mysqld']['myisam-recover-options'] = 'BACKUP'
+default['mysqld']['my.cnf']['mysqld']['myisam_recover_options'] = 'BACKUP'
```

Comment:
Oops I forgot to mention that. Both docs (Mysql and MariaDB) mention this attribute with underscores in their docs. So I changed it just to be sure. But since other attributes in the defaults.rb that are written with dashes that appear in the docs with underscores work fine, I'm sure you're right. It was an unnecessary change.